frontend/src/components/datasets/Dataset.tsx

```diff
@@ -280,6 +281,13 @@ export const Dataset = (): JSX.Element => {
 							disabled={false}
 						/>
 					</Tabs>
+					{folderId !== null ?
+							<Box>
```

Comment:
Would aligning with the file table will make it look cleaner? Thoughts?

<img width="922" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/253a63e4-233b-441e-8aec-e6880243b346">


Add 
```
 padding: "24px 24px 0 24px"
```

Reply:
@longshuicy 

I added this but somehow it's not aligned properly, but it looks like it's okay on the Files component. Not sure what I did wrong.